### DISCUSSION TITLE
Calling different resolvers within a query

### DISCUSSION THREAD

**Discussion by lasongrzegorz** [mirumee/ariadne#1128] [Q&A]
Hi,
I would like to split calls to different resolvers to manage memory. Imagin I am having such a schema :
```
query = QueryType()

type Query {
  ping: Pong
}

type Pong {
  pong: String
  pingPong: String
  anotherPingPong: String
}
```
I would like each `Pong` field to be calling different resolver, so that I do not call `ping` resolver each time when accessing pong, pingPong or anotherPingPong. 

I would imagine it would be something like below but obviously I am missing some code there. Is that at all possible? 
```
query = QueryType()
...???...  # how to bind these to query? 
query.set_field("pong", resolve_pong)
query.set_field("pingPong", resolve_ping_pong)
query.set_field("anotherPingPong", resolve_another_ping_pong)
```

Thanks a lot in advance

**Comment by rafalp** [comment]
Hello.

Please see [this document](https://ariadnegraphql.org/docs/resolvers). It shows exactly what you are asking for.
